tests/tests_lite/run_standalone_tasks.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+# THIS FILE ASSUMES IT IS RUN INSIDE THE tests/tests_lite DIRECTORY
+
+# needs to run outside of `pytest`
+echo "Running utilities/test_warnings.py"
+python utilities/test_warnings.py
```

Comment:
@awaelchli You need to remove this again after #14556 is merged (and delete from Lite GPU CI job)